spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -18,7 +18,7 @@
 // Contributors:
 // John D. Ament
 
-[[release_notes_11]]
+[[release_notes]]
```

Comment:
Please leave the section title in there, and add a new section title `[[release_notes_12]]`

Reply:
done